cedar-policy-validator/src/schema.rs

```diff
@@ -842,7 +856,10 @@ impl ValidatorSchema {
     pub fn action_entities(&self) -> std::result::Result<Entities, EntitiesError> {
         let extensions = Extensions::all_available();
         Entities::from_entities(
-            self.action_entities_iter(),
+            self.actions
+                .values()
+                .map(|entity| entity.as_ref().clone())
+                .collect::<Vec<Entity>>(),
```

Comment:
Should be able to drop this  `collect` since `from_entities` just wants `impl IntoIterator<...>`